components/eam/src/chemistry/aerosol/wetdep.F90

```diff
@@ -5,19 +5,21 @@ module wetdep
 ! Wet deposition routines for both aerosols and gas phase constituents.
 ! 
 !-----------------------------------------------------------------------
+#include "../yaml/common_files/common_uses.ymlf90"
 
 use shr_kind_mod, only: r8 => shr_kind_r8
 use ppgrid,       only: pcols, pver
 use physconst,    only: gravit, rair, tmelt
 use cam_abortutils, only: endrun
 use mam_support,  only: min_max_bound
+use spmd_utils,   only : masterproc
 
 implicit none
 save
 private
 
 public :: wetdepa_v2  ! scavenging codes for very soluble aerosols -- CAM5 version
-public :: clddiag     ! calc of cloudy volume and rain mixing ratio
+!public :: clddiag     ! calc of cloudy volume and rain mixing ratio
```

Comment:
Just wonder why it is commented out.  Not long used in other modules?

Reply:
That's right. I will just remove this line.